src/shared/ui/survey/tests/DatePickerItem.test.tsx

```diff
@@ -36,8 +38,15 @@ describe('Test DatePickerItem', () => {
 
     const resultProp = format(datePicker.props.value, 'yyyy-MM-dd');
 
-    const expectedDate = format(new Date(0), 'yyyy-MM-dd');
-
+    const expectedDate = formatDateToUTC(getDateZero);
     expect(resultProp).toBe(expectedDate);
   });
 });
+
+const formatDateToUTC = (date: Date): string => {
+  const year = date.getUTCFullYear();
+  const month = String(date.getUTCMonth() + 1).padStart(2, '0');
+  const day = String(date.getUTCDate()).padStart(2, '0');
+
+  return `${year}-${month}-${day}`;
+};
```

Comment:
Can you elaborate why you're doing custom formatting here instead of using `format(new Date(0), 'yyyy-MM-dd')`

Reply:
Of course, the test is waiting for the result which is date 0 of the -8 time zone, so I could deal with two situations, using a library called date-fns-timezone or creating a function for it, so I decided to do it manually.

Reply:
the difference is when you use  format(new Date), it was returned 1969, so this function was to equalize the  date by time zonee